tests/zbd/010

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+# SPDX-License-Identifier: GPL-3.0+
+# Copyright (C) 2022 Google LLC
+
+. tests/zbd/rc
+. common/null_blk
+. common/scsi_debug
+
+DESCRIPTION="test gap zone support with F2FS"
+QUICK=1
+
+requires() {
+	_have_fio
+	_have_kernel_option BLK_DEV_ZONED
```

Comment:
This check for BLK_DEV_ZONED is not required here, since it is done in group_requires() in tests/zbd/rc.

Reply:
That check has been removed.